src/json_interface.c

```diff
@@ -70,11 +65,12 @@ int ji_init(const char *file_name)
     read_size = getline(&buf, &length, file_handle);
     while( -1 != read_size ) {
         char *c = strchr(buf, COMMA);
```

Comment:
If for some reason the last flush mostly happened, we should protect & bail vs. segfault by dereferencing NULL.